packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -1514,6 +1514,7 @@ describe('plugin-meetings', () => {
           meeting.webex.internal.mercury.on = sinon.stub().returns(true);
           meeting.setReconnectListener(test1, test2);
           assert.instanceOf(meeting.reconnectionManager, ReconnectionManager);
+          assert.calledTwice(meeting.webex.internal.mercury.on);
```

Comment:
The `it` block for this test doesn't describe what it is testing at all. Let's change it to:

```js
it('should listen to mercury events', () => {
```